matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/space/DefaultSpaceService.kt

```diff
@@ -77,7 +78,7 @@ internal class DefaultSpaceService @Inject constructor(
             if (isPublic) {
                 this.roomAliasName = roomAliasLocalPart
                 this.powerLevelContentOverride = (powerLevelContentOverride ?: PowerLevelsContent()).copy(
-                        invite = 0
+                        invite = if (isPublic) 0 else Role.Moderator.value
```

Comment:
Can you use `Role.Default.value` instead of `0`?

Reply:
Done